docs/topics/arrays.md

```diff
@@ -31,32 +50,244 @@ fun main() {
 ```
 {kotlin-runnable="true" kotlin-min-compiler-version="1.3"}
```

Comment:
It's a good point. I've updated the content so that all samples are runnable except for the `emptyArray()` because it could confuse readers when nothing is printed.